xbmc/pvr/PVRGUIInfo.h

```diff
@@ -172,7 +172,9 @@ namespace PVR
     bool                            m_bIsPlayingRecording;
     bool                            m_bIsPlayingEncryptedStream;
     bool                            m_bHasTVChannels;
+    bool                            m_bHasTVHiddenChannels;
```

Comment:
Do you think you could change the order of the words, i.e. "HasHiddenTVChannels" instead of "HasTVHiddenChannels"?